lua/luasnip/extras/postfix.lua

```diff
@@ -55,6 +55,13 @@ local function generate_opts(match_pattern, user_callback)
 	}
 end
 
+local function wrap_condition(user_condition, trigger, match_pattern)
+	return function(line_to_cursor, ...)
+		return (not user_condition or user_condition(line_to_cursor, ...))
+			and line_to_cursor:gsub(trigger, ""):match(match_pattern) ~= nil
```

Comment:
This `:gsub` might now work as expected. It'll treat `trigger` as lua pattern.

I think replacing it with `:sub` makes more sense.